extensions/wikia/CoppaTool/RenameIPProcess.class.php

```diff
@@ -0,0 +1,470 @@
+<?php
+
+class RenameIPProcess {
+
+	const RENAME_TAG = 'renamed_to';
+	const PROCESS_TAG = 'rename_in_progress';
+	const PHALANX_BLOCK_TAG = 'phalanx_block_id';
+
+	const MAX_ROWS_PER_QUERY = 500;
+
+	const LOG_STANDARD = 'standard';
+	const LOG_BATCH_TASK = 'task';
+	const LOG_OUTPUT = 'output';
+
+	// Define what needs changing in core MW tables
+	/*
+	 * Task definition format:
+	 *	 'table' => (string) table name
+	 *   'userid_column' => (string) column name with user ID or null if none
+	 *   'username_column' => (string) column name with user name
+	 *   'conds' => (array) additional conditions for the query
+	 */
+	/**
+	 * Stores the predefined tasks to do for global (wikicities) database.
+	 * Here should be mentioned all core tables not connected to any extension.
+	 *
+	 * @var $mGlobalDefaults array
+	 */
+	static private $mGlobalDefaults = array(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$mGlobalDefaults" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)

[project-id]: wikia:app